src/Tribe/Attendees.php

```diff
@@ -307,22 +307,32 @@ public function enqueue_assets( $hook ) {
 
 		add_thickbox();
 
-		$mail_data = array(
+		$config_data = [
 			'nonce'           => wp_create_nonce( 'email-attendee-list' ),
 			'required'        => esc_html__( 'You need to select a user or type a valid email address', 'event-tickets' ),
 			'sending'         => esc_html__( 'Sending...', 'event-tickets' ),
 			'ajaxurl'         => admin_url( 'admin-ajax.php' ),
 			'checkin_nonce'   => wp_create_nonce( 'checkin' ),
 			'uncheckin_nonce' => wp_create_nonce( 'uncheckin' ),
 			'cannot_move'     => esc_html__( 'You must first select one or more tickets before you can move them!', 'event-tickets' ),
-			'move_url'        => add_query_arg( array(
+			'move_url'        => add_query_arg( [
 				'dialog'    => Tribe__Tickets__Main::instance()->move_tickets()->dialog_name(),
 				'check'     => wp_create_nonce( 'move_tickets' ),
 				'TB_iframe' => 'true',
-			) ),
-		);
+			] ),
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 6470831bd29e2b2f70ee457b59d881eb</sub>